manifests/server/read_database_ini.pp

```diff
@@ -62,7 +63,7 @@
       $subprotocol = 'postgresql'
 
       $subname = $database_ssl ? {
-        true    => "//${database_host}:${database_port}/${database_name}?ssl=true",
+        true    => "//${database_host}:${database_port}/${database_name}?ssl=true${database_subparam}",
```

Comment:
In some of our other modules we use the following pattern for this
```
 $jdbc_ssl_properties   = "?ssl=true&sslfactory=org.postgresql.ssl.jdbc4.LibPQFactory&sslmode=verify-full&sslrootcert=${localcacert}"
```
That is to say maybe instead of the `database_ssl` bool there should just be a string args like `jdbc_ssl_properties` which we construct the subname from and `database_ssl` just equals `jdbc_ssl_properties == ''`.